README.md

```diff
@@ -12,7 +12,8 @@ The Elixir/Phoenix adapter for [Inertia.js](https://inertiajs.com/).
 - [Validations](#validations)
 - [Flash messages](#flash-messages)
 - [CSRF protection](#csrf-protection)
-- [Server-side rendering](#server-side-rendering-experimental)
+- [Testing](#csrf-protection)
```

Comment:
Only minor thing: The link is wrong here, should be `#testing`.

Reply:
Whoops! Thanks @rverton :)